src/App.jsx

```diff
@@ -14,70 +15,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood]
+    = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+        {selectedGood && (
+          <button
+            onClick={() => setSelectedGood('')}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          >
+            {}
+          </button>
+        )}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              data-cy="Good"
+              className={selectedGood === good
+                && 'has-background-success-light'}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  onClick={() => (
+                    selectedGood === good ? setSelectedGood()
+                      : setSelectedGood(good)
+                  )
+                  }
+                  data-cy={good === selectedGood ? 'RemoveButton' : 'AddButton'}
```

Comment:
To avoid code duplication you can create a variable inside map function and use it here

`const isGoodSelected = good === selectedGood`